apps/app.go

```diff
@@ -66,11 +66,18 @@ type ListedApp struct {
 	Labels    []model.MarketplaceLabel `json:"labels,omitempty"`
 }
 
-const MaxAppID = 32
+const (
+	MinAppIDLength = 3
+	MaxAppIDLength = 25
```

Comment:
Why 25? And the comment says, up to 20?

Reply:
I wasn't aware of the fact that the app id is not longer used directly. With that in mind I'm happy to revert back to 32 bytes. What do you think @levb ?

Reply:
@hanzei should work.